src/lib/sw_engine/tvgSwStroke.cpp

```diff
@@ -382,9 +382,16 @@ static void _lineTo(SwStroke& stroke, const SwPoint& to)
     if (delta.zero()) return;
 
     //compute length of line
-    auto lineLength =  mathLength(delta);
     auto angle = mathAtan(delta);
 
+    /* The lineLength is used to determine the intersection of strokes outlines.
+       The scale needs to be reverted since the stroke width has not been scaled.
+       An alternative option is to scale the width of the stroke properly by
+       calculating the mixture of the sx/sy rating on the stroke direction. */
```

Comment:
I may misunderstand the comment - isn;t the alternative option that you have mentioned done in 395-397?

Reply:
No, I mean here we can scale the stroke width instead reverting lineLength scale. The problem is, stroke width is a scalar value but we have two axis scale factors from transform matrix which means we could scale it alternatively by computing the total scale value from sx, xy by measuring the direction of the stroke.